src/nvim/autocmd.c

```diff
@@ -137,12 +137,6 @@ static inline const char *get_deleted_augroup(void) FUNC_ATTR_ALWAYS_INLINE
 // Show the autocommands for one AutoPat.
 static void aupat_show(AutoPat *ap)
 {
-  // Check for "got_int" (here and at various places below), which is set
-  // when "q" has been hit for the "--more--" prompt
-  if (got_int) {
-    return;
-  }
-
   // pattern has been removed
   if (ap->pat == NULL) {
```

Comment:
This check may also have to be moved, but I haven't found a problem caused by this check yet.